### PR TITLE
Support local testing.

### PR DESCRIPTION
**Pull request by tomwhite**
This isn't quite ready since it changes DownloadDatasetParallelTask to DownloadDatasetTask. Implementing https://github.com/bigdatagenomics/eggo/issues/21 is probably the best way to fix this.

### CODE REVIEW

**Comment by tomwhite** [comment]
I implemented #21using Hadoop streaming. This works locally but not on the EC2 Spark cluster since there is no MR cluster running :)

**Comment by fnothaft** [comment]
@tomwhite you should be able to start MR on the a spark-ec2 cluster with `./ephemeral-hdfs/bin/start-all.sh`, IIRC.

**Comment by tomwhite** [comment]
Thanks @fnothaft - that works. I'll do some more testing and update this PR.

**Comment by laserson** [comment]
Yes, the eggo scripts should already be starting MR on EC2.

File: bin/download_upload.sh [hunk 1]
```diff
@@ -19,7 +19,9 @@
 # COMPRESSION_TYPE can be NONE or GZIP
 
 # download the file locally
-source /root/eggo/eggo-ec2-variables.sh
+if [ -f /root/eggo/eggo-ec2-variables.sh ]; then
+  source /root/eggo/eggo-ec2-variables.sh
+fi
```
Comment (laserson) [inline bin/download_upload.sh:24]: We may be able to eliminate this script entirely by using the Luigi Hadoop API.  I'll wait till we merge this PR before I try to refactor.

**Comment by tomwhite** [comment]
I see that fabric_util.py calls `/root/ephemeral-hdfs/bin/start-all.sh`, but MR didn't seem to be running. I'll have a look at that. I also need to fix a problem with the streaming job input file being on the local fs, not HDFS.

**Comment by laserson** [comment]
let's just put the tmp command file in hdfs

**Comment by tomwhite** [comment]
The reason that MR was not running was because I hadn't run `fab setup_master` that time. Opened https://github.com/bigdatagenomics/eggo/issues/26 to provide a single start command.

I tested this change on a cluster and it works, so I think it's ready to merge now.

**Pull request merged**